runelite-client/src/main/java/net/runelite/client/plugins/runepouch/RunepouchOverlay.java

```diff
@@ -99,6 +99,8 @@ public Dimension render(Graphics2D graphics, java.awt.Point point)
 		graphics.setFont(FontManager.getRunescapeSmallFont());
 
 		StringBuilder tooltipBuilder = new StringBuilder();
+		boolean empty = true;
```

Comment:
Maybe better than using dummy boolean like this would be instead checking for tooltipBuider.toString().isEmpty()